src/styles/blocks/now-on-view.scss

```
@@ -0,0 +1,17 @@
+.now-on-view {
```

it is according to BEM?
try to name classes according to the BEM
https://ru.bem.info/methodology/
```suggestion
.now-on-view {
```